### ISSUE TITLE
Web UI calls /api/environments?status=open&status=ack and receives a 500 error

### ISSUE THREAD

**Issue by jurgenhaas** [alerta/alerta#1250]
**Issue Summary**

Updated to 8.0 on a docker based installation and all seems to be working just fine, except that the web ui is regularely (every few seconds) calling `/api/environments?status=open&status=ack` and always receives a 500 error message. When watching the logs on the server, this is what I see:

```
2020-06-23 06:37:13,306 DEBG 'uwsgi' stdout output:
2020-06-23 06:37:13,306 flask.app[24]: [ERROR] 'arocom' request_id=3be681fd-59ac-497f-bc48-398bad676004 ip=94.79.163.37
Traceback (most recent call last):
  File "/venv/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/venv/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/venv/lib/python3.7/site-packages/flask_cors/decorator.py", line 128, in wrapped_function
    resp = make_response(f(*args, **kwargs))
  File "/venv/lib/python3.7/site-packages/alerta/auth/decorators.py", line 89, in wrapped
    return f(*args, **kwargs)
  File "/venv/lib/python3.7/site-packages/alerta/models/metrics.py", line 258, in wrapped
    response = f(*args, **kwargs)
  File "/venv/lib/python3.7/site-packages/alerta/utils/response.py", line 18, in decorated
    return func(*args, **kwargs)
  File "/venv/lib/python3.7/site-packages/alerta/views/alerts.py", line 473, in get_environments
    environments = Alert.get_environments(query)
  File "/venv/lib/python3.7/site-packages/alerta/models/alert.py", line 516, in get_environments
    return db.get_environments(query)
  File "/venv/lib/python3.7/site-packages/alerta/database/backends/postgres/base.py", line 613, in get_environments
    } for e in environments]
  File "/venv/lib/python3.7/site-packages/alerta/database/backends/postgres/base.py", line 613, in <listcomp>
    } for e in environments]
KeyError: 'arocom'

2020-06-23 06:37:13,308 DEBG 'uwsgi' stdout output:
[pid: 24|app: 0|req: 3321/7117] 172.21.0.1 () {58 vars in 3843 bytes} [Tue Jun 23 06:37:13 2020] GET /api/environments?status=open&status=ack => generated 1565 bytes in 7 msecs (HTTP/1.1 500) 5 headers in 205 bytes (1 switches on core 0)
```

The key value (`arocom` in this example) varies and is always one of my properly configured environment values.

**Environment**
- OS: Ubuntu 18.04 LTS
- API version: 8.0.0
- Deployment: Docker
- Database: Docker image tagged latest

- Server config:
  Auth enabled? Yes
  Auth provider? GitLab
  Customer views? Yes

- web UI version: 8.0.0
- CLI version: 7.5.6

**Comment by satterly** [comment]
Thanks for reporting. Should be fixed now. Will release later today. 👍

**Comment by satterly** [comment]
Released as version 8.0.1
